core/src/main/kotlin/ca/allanwang/kau/utils/ColorUtils.kt

```diff
@@ -66,6 +66,19 @@ val Int.isColorTransparent: Boolean
     get() = Color.alpha(this) != 255
 
 @ColorInt
+fun Int.blendWith(@ColorInt color:Int, @FloatRange(from = 0.0, to = 1.0) ratio:Float):Int {
+    val inverseRatio = 1f - ratio
+    val a = Color.alpha(this) * inverseRatio + Color.alpha(color) * ratio
+    val r = Color.red(this) * inverseRatio + Color.red(color) * ratio
+    val g = Color.green(this) * inverseRatio + Color.green(color) * ratio
+    val b = Color.blue(this) * inverseRatio + Color.blue(color) * ratio
+    return Color.argb(a.toInt(), r.toInt(), g.toInt(), b.toInt())
+}
+
+@ColorInt
+fun Int.withAlpha(@FloatRange(from = 0.0, to = 1.0) alpha:Float) = withAlpha((255 * alpha).toInt())
```

Comment:
I don't want to add this because it may easily be confused with the int version. Plus withAlpha implies that the direct value (int) is given. Even when customizable, sliders use ints, so I think it should be okay